intake/interface/source/defined_plots.py

```diff
@@ -54,38 +59,62 @@ class Plots(BaseView):
     """
     select = None
 
-    def __init__(self, source=None, **kwargs):
-        self.custom = pn.widgets.Button(name='Customize...', width_policy='min', align='center')
+    def __init__(self, source=None, edit_callback=None, **kwargs):
+        self.custom = pn.widgets.Button(name='Create', width=70, align='center')
         self.source = source
+        self.callback_on_edit = edit_callback
         self.panel = pn.Column(name='Plot', width_policy='max', margin=0)
+        self._behavior_callback = None
         super().__init__(**kwargs)
 
     def setup(self):
         self.instructions = pn.pane.Markdown(
-            self.instructions_contents, align='center', width_policy='max')
+            "**Select Plot**", align='center', width=70)
```

Comment:
Will need to test this on a range of browser sizes if using any absolute numbers for width.

Reply:
I tested it at various zoom levels and browsers (Chrome, Firefox, Brave) and it looked correct on all those.
In the worst case, the text will simply wrap and display on two lines, so I think this is a minor concern.